docs/site/translation/getting-started.md

```diff
@@ -6,19 +6,17 @@ title: Getting Started
 
 Before getting started to contribute data in CLDR, and jumping in to using the Survey Tool, it is important that you understand the CLDR process.
 
-1. Understand the basics about the CLDR process and the community-based model and the Technical Committee. Read: [CLDR Process](/index/process)
+1. Understand the basics about the CLDR process and the community-based model and the Technical Committee. Read: [CLDR Process](../../index/process)
 2. Read the Getting Started topics on the Information Hub:
-    - [Survey Tool Guide](/translation/getting-started/guide)
-    - [Survey Tool Stages](/translation/getting-started/survey-tool-phases)
-    - [Handling Errors and Warnings](/translation/getting-started/errors-and-warnings)
-    - [Handling Logical Group Errors](/translation/getting-started/resolving-errors)
-    - [Plurals & Units](/translation/getting-started/plurals)
-    - [Review Date & Time](/translation/date-time)
-    - [Review Numbers](/translation/number-currency-formats)
-    - [Review Zones](/translation/time-zones-and-city-names)
-    - [Data stability](/translation/getting-started/data-stability)
-    - [Empty cache](/translation/getting-started/empty-cache)
-
-\*If you (individual or your organization) have not established a connection with the CLDR technical committee, start with [Survey Tool Accounts](/index/survey-tool/survey-tool-accounts).
-
+    - [Survey Tool Guide](guide)
```

Comment:
This doesn't work, as far as I can tell. 
It goes to 
https://github.com/unicode-org/cldr/blob/btangmu/t08038_b/docs/site/translation/guide
but it needs to go to 
https://github.com/unicode-org/cldr/blob/btangmu/t08038_b/docs/site/translation/getting-started/guide.md

That is, the .md files that correspond to a directory are at the same level as that directory, not inside it.

Same problem with various other links.

Reply:
You're right. Changed to /translation/getting-started/guide